dev/breeze/src/airflow_breeze/utils/parallel.py

```diff
@@ -458,6 +458,8 @@ def check_async_run_results(
             for output in outputs:
                 Path(output.file_name).unlink(missing_ok=True)
 
+    return tests_failed
```

Comment:
The right solution here will be:

```
if tests_failed:
   sys.exit(1)
```

The problem is that we do not check the result of `check_async_run_results` - the method was supposed to "sys.exit(1)` here rather than return False.

My bad I have not noticed it :(